packages/babel-node/src/_babel-node.ts

```diff
@@ -64,6 +68,21 @@ const replPlugin = ({ types: t }: PluginAPI): PluginObject => ({
           );
         }
       }
+
+      if (hasTopLevelAwait) {
+        const body = path.node.body;
+        for (let i = body.length - 1; i >= 0; i--) {
+          if (t.isExpressionStatement(body[i])) {
```

Comment:
This doesn't work in cases like `0; { 1; }`. We need to use `path.getCompletionRecords`

Reply:
https://github.com/nodejs/node/blob/22ac5976a837e4caab7c3c8f92b211fe2ba29076/lib/internal/repl/await.js#L224-L252
Nodejs does not support this either. Do we want to unify with it?

Reply:
Yes, let's unify with Node.js :)